extras/envoy-xds/sample-config.json

```diff
@@ -1,23 +1,27 @@
 {
+    "discoveryServerAddr": "multi:///host1:port,host2:port,host3:port",
+    "dc-id": "default",
+    "database": "default",
```

Comment:
Isn't it possible that each shard uses a different database name? How would that be handled in that case? The same is with dc-id. An app configuration could use a remote shard also.

Reply:
yea so current envoy config is for all shards within a database and dc. 

in current version 
1. each DC requires individual envoy xds server. thus each DC specific envoy XDS will have the dc-id configured accordingly. this also means that clients won't be able to make cross DC calls directly.
2. we will only support 1 database for now. 

I think above requirements are fine for current DKV. 

in next version, we will change both DKV client SDK and envoy xds to support multi database, inter DC interaction with the same envoy xds and DKV cluster. 

above is additional effort which we can prioritise based on which client is onboarding.